tools/azure-sdk-tools/devtools_testutils/aio/proxy_testcase_async.py

```diff
@@ -0,0 +1,54 @@
+from ..proxy_testcase import (
+    get_test_id,
+    start_record_or_playback,
+    transform_request,
+    stop_record_or_playback,
+)
+
+from azure.core.pipeline.transport import AioHttpTransport
+
+from azure_devtools.scenario_tests.utilities import trim_kwargs_from_test_function
+
+def RecordedByProxyAsync(func):
```

Comment:
Nit: have we applied `black` on all the new files?

Reply:
We haven't, but I should -- thanks for the reminder!